src/v2/Components/__stories__/Molecules.story.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  Box,
+  Text,
+  GridColumns,
+  Column,
+  Button,
+  Spacer,
+  ProgressDots,
+} from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import React from "react"
+import styled from "styled-components"
+import { FullBleed } from "../FullBleed"
+
+export default {
+  title: "Molecules/Kitchen Sink",
+  parameters: {
+    layout: "fullscreen",
+  },
+}
+
+const AppContainer = styled(Box)`
+  max-width: ${themeGet("breakpoints.lg")};
```

Comment:
We probably are going to want to deprecate all the token helpers like `color/space/breakpoint` etc... Since they can't get at the theme context they are all tied to the v2 theme. We could make some hooks like useSpace, etc, and inside of styled-components, a simple `themeGet` suffices.

Reply:
Though in general I think it's kind of rare to actually *need* something like `color` or `space` inside of a component. It tends to just get reached for by mistake I think.

Reply:
Agree that it's relatively rare. Can happen when needing to add things outside a styled prop though, which happens on occasion. I'm 👍 on just recommending themeGet as a replacement.